### DISCUSSION TITLE
What's the perfect beverage to drink while coding a Next app?

### DISCUSSION THREAD

**Discussion by sorcerer-worker** [vercel/next.js#40199] [Polls]
Yeah. Literally a poll about the perfect beverage to drink while coding a Next app. Nothing more nothing less. Given as much as I occasionally read issues on here I rarely see actual community-like behavior besides the random ideas.

**Comment by gaston-flores** [comment]
For me, it's [Yerba mate 🧉 ](https://en.wikipedia.org/wiki/Mate_(drink))

**Reply by sorcerer-worker** [reply]
> For me, it's [Yerba mate mate ](https://en.wikipedia.org/wiki/Mate_(drink))

Woah. I've never seen that before I'll have to try it one of these days.

**Reply by fzinnah** [reply]
Love me some Yerba Mate! cheers to that! 🧉

**Comment by adxmcollins** [comment]
Probably super boring, but water 🤷‍♂️

**Reply by sorcerer-worker** [reply]
Never would've seen that answer coming. Well at least you're staying hydrated. lol

**Comment by anshuman-8** [comment]
well..Water is fine

**Comment by thelovesmith** [comment]
Un Doppio Espresso for me
